test/variables/static/multilocale.chpl

```diff
@@ -0,0 +1,14 @@
+proc add(arg) {
+  @functionStatic(sharingKind.computePerLocale)
+  ref counter = 0;
+
+  counter += arg;
+  return counter;
+}
+
+coforall loc in Locales do on loc {
```

Comment:
I think you can make this a ``for`` loop and get rid of the prediff. It doesn't seem like the ``coforall`` is exercising any important behavior here.